server/src/core/odoo.rs

```diff
@@ -468,8 +469,10 @@ impl SyncOdoo {
     fn build_modules(session: &mut SessionInfo) {
         {
             let addons_symbol = session.sync_odoo.get_symbol(session.sync_odoo.config.odoo_path.as_ref().unwrap(), &tree(vec!["odoo", "addons"], vec![]), u32::MAX)[0].clone();
-            let addons_path = addons_symbol.borrow().paths().clone();
-            for addon_path in addons_path.iter() {
+            let SymbolPath::Multiple(addons_paths) = addons_symbol.borrow().path() else {
+                panic!("Odoo addons symbol paths should be a namespace");
+            };
```

Comment:
@fda-odoo we wrote this together.
But since then `SymbolPath::as_multiple` has been introduced.
Shall I replace this by simply `let addon_paths = addons_symbol.borrow().path().as_multiple()`?